package.json

```diff
@@ -29,10 +31,9 @@
   },
   "engineStrict": true,
   "files": [
-    "**/*.js",
-    "!/test.js",
-    "/Readme.md",
-    "/License.md"
+    "lib/index.js",
+    "lib/src/**/*.js",
+    "bin/elm-live.js"
```

Comment:
It is probably a good idea to publish the license to npm. I would suggest adding it back to the list of files here. :)

Reply:
Lol def did not mean to permanently remove that! Thanks for the check in!